src/app/drive/components/ItemDetailsDialog/ItemDetailsDialog.tsx

```diff
@@ -68,7 +68,11 @@ const ItemsDetails = ({ item, translate }: { item: ItemDetailsProps; translate:
  * - Location
  *  */
 
-const ItemDetailsDialog = ({ onSharedItemClicked }: { onSharedItemClicked?: (item: AdvancedSharedItem) => void }) => {
+const ItemDetailsDialog = ({
+  onButtonClicked,
+}: {
+  onButtonClicked: (item: AdvancedSharedItem | DriveItemData, event?) => void;
```

Comment:
Add more descriptive name to the callback in order to know what is the behaviour that has to accomplish